Integrations/integration-DemistoLocking.yml

```diff
@@ -0,0 +1,153 @@
+description: Locking mechanism that prevents concurrent execution of different tasks
+detaileddescription: "The Demisto Locking integration allows prevention of concurrent
+  execution of scripts or commands, using a wait-lock-release flow (mutex). \nUse
+  the lock name argument to support multiple locks in different flows."
+configuration:
+- display: Default timeout for wait on locks to be released
+  name: timeout
+  defaultvalue: "60"
+  type: 0
+  required: true
+script:
+  script: |
+    function guid() {
+      function s4() {
+        return Math.floor((1 + Math.random()) * 0x10000)
+          .toString(16)
+          .substring(1);
+      }
+      return s4() + s4() + '-' + s4() + '-' + s4() + '-' + s4() + '-' + s4() + s4() + s4();
+    }
+
+    var setLock = function(guid, info) {
+        var integrationContext = getIntegrationContext();
```

Comment:
can it be null?
If so, change to:
`var integrationContext = getIntegrationContext() || {};`